custom/icds_reports/sqldata/base_operationalization.py

```diff
@@ -35,3 +39,42 @@ def rows(self):
                     0
                 ]
             ]
+
+
+class BaseOperationalizationBeta(ICDSMixin):
```

Comment:
this can probably inherit from `BaseOperationalization` and only override `rows`

Reply:
yeah that makes sense.